test/gc/test-net-timeout.js

```diff
@@ -35,7 +35,11 @@ function getall() {
   if (count >= todo)
     return;
 
-  const req = net.connect(server.address().port, server.address().address);
+  if (common.hasIPv6)
+   var host = '::1';
+  else
+    var host = '127.0.0.1';
```

Comment:
@bnoordhuis do we need to specify a port (i.e. `server.listen(common.PORT, common.localhostIPv4, getall);`)?

```
/tmp/gib/node/test/gc/test-net-timeout.js:38
  const req = net.connect(server.address().port, server.address().address);
                                          ^

TypeError: Cannot read property 'port' of null
    at getall (/tmp/gib/node/test/gc/test-net-timeout.js:38:43)
```